components/console/helpers/tree.rst

```diff
@@ -105,19 +105,30 @@ You can build a tree by creating a new instance of the
     use Symfony\Component\Console\Helper\TreeHelper;
     use Symfony\Component\Console\Helper\TreeNode;
 
-    $node = TreeNode::fromValues([
-        'Command',
-        'Controller' => [
-            'DefaultController.php',
-        ],
-        'Kernel.php',
-    ]);
```

Comment:
TreeNode from multidimensional array should be documented I think.
(I build this array first, and then use it in TreeHelper)